[DOC] Some minor typos fixed in tutorials (#2917)
seqan/seqan3: doc/tutorial/sequence_file/index.md

## Applying a filter to a file

> On some occasions you are only interested in sequence records that fulfill a certain criteria,

Reviewer: ```suggestion
On some occasions, you are only interested in sequence records that fulfill a certain criteria,
```

## Writing records

The easiest way to write to a sequence file is to use the seqan3::sequence_file_output::push_back() member function.

> It works similarly to how it works on an std::vector.

Reviewer 2: We use a for std. See here: https://github.com/seqan/seqan3/wiki/Coding-guideline#Namespaces

Again we want to point out the convenient advantage of modelling files as ranges. The seqan3::sequence_file_input models
std::ranges::input_range and in the "reading a file" section you already saw a few examples of how to pipe a view onto

> such a range. The output files model std::ranges::output_range and with that, we can use ranges algorithm like

Reviewer: ```suggestion
such a range. The output files model std::ranges::output_range. This allows us to use ranges algorithms like
```
